app/helpers/transaction_type_helper.rb

```diff
@@ -25,4 +25,25 @@ def sort_category_column_cell_vacant(object, transaction_type)
       content:,
     )
   end
+
+  def anwser_for_transaction_type(legal_aid_application:, transaction_type:)
+    total = legal_aid_application.transactions_total_by_category(transaction_type.id)
+    has_transaction_type = legal_aid_application.has_transaction_type?(transaction_type)
+
+    if has_transaction_type && total.zero?
+      if legal_aid_application.uploading_bank_statements?
+        t("generic.yes")
+      else
+        "Yes, but none specified"
+      end
```

Comment:
could these be rewritten as single line conditionals? I personally think they are easier to read than the nested 'if' statements but I think it is a matter of opinion so up to you

Reply:
```
legal_aid_application.uploading_bank_statements? ? t("generic.yes") : t("generic.yes_but_none")
```

Reply:
Also, I think the content here ("yes, but none specified") could perhaps be put in a .yml file instead?

Reply:
done